eng/test-configuration.json

```diff
@@ -3,15 +3,12 @@
   "defaultOnFailure": "fail",
   "localRerunCount" : 3,
   "retryOnRules": [
-    {"testName": {"contains": "AppOfflineDroppedWhileSiteStarting_SiteShutsDown_InProcess"}},
-    {"testName": {"contains": "AppOfflineDroppedWhileSiteIsDown_SiteReturns503_InProcess"}},
-    {"testName": {"contains": "CheckFrebDisconnect"}},
-    {"testName": {"contains": "CheckStdoutWithLargeWrites"}},
-    {"testName": {"contains": "ServerShutsDownWhenMainExitsStress" }},
-    {"testName": {"contains": "AddressRegistrationTests" }},
+    {"testName": {"contains": "FlakyTestToEnsureRetryWorks" }},
     {"testName": {"contains": "MaxRequestBufferSizeTests.LargeUpload" }},
     {"testName": {"contains": "ReaderThrowsResetExceptionOnInvalidBody" }},
-    {"failureMessage": {"contains":"System.AggregateException : One or more errors occurred. (Operation did not succeed after 15 retries (Site is started but no worker process found) (Site is started but no worker process found)"}},
+    {"testAssembly": {"contains": "IIS"}},
+    {"testAssembly": {"contains": "ShadowCopy"}},
```

Comment:
The `retryOnRules` inner rules area `OR`'d together unless you do something like `{"testAssembly": {"contains": "IIS"}, "failureMessage": "yada yada"}}` in one of them (the two properties are `AND`ed in that case). So, these lines will retry every IIS test assembly and _also_ every shadow copy test assembly.

Reply:
Correct, I want to retry every IIS functional or every ShadowCopy, I guess I don't necessarily need to specify shadow copy here since I think they have an IIS in the name

Reply:
suggest leaving out ShadowCopy then because it covers a subset

Reply:
note that's probably a nit but remains my preference